octobot_trading/exchanges/exchange_manager.py

```diff
@@ -158,6 +159,7 @@ def reset_exchange_personal_data(self):
         try:
             await self.exchange.modify_channels()
             await self.exchange.create_backtesting_exchange_producers()
+            await start_backtesting(self.exchange.backtesting)
```

Comment:
Why this is not made in `ExchangeSimulator` ? Maybe an `ExchangeSimulator.start()` or `.start_backtesting()` ?